src/term/mod.rs

```diff
@@ -640,7 +640,7 @@ pub struct Term {
     alt: bool,
 
     /// The cursor
-    cursor: Cursor,
+    pub cursor: Cursor,
```

Comment:
Rather than making cursor `pub`, I think we should add an accessor that returns a read-only ref. Something like

```rust
fn cursor(&self) -> &Cursor {}
```